### ISSUE TITLE
LJ Speech failed to alignment with the latest version.

### ISSUE THREAD

**Issue by begeekmyfriend** [Rayhane-mamah/Tacotron-2#147]
I have tried the latest version and it still did not achieve alignment at 15K steps with the default dataset LJ Speech 1.1. What modified in `hparam.py` was only `predict_linear = False`. It seemed over fitting since the loss was low.
![step-15000-align](https://user-images.githubusercontent.com/6031938/43992297-e9d1a238-9daf-11e8-9ff2-1fe76551c2ed.png)
![screenshot from 2018-08-11 21-49-47](https://user-images.githubusercontent.com/6031938/43992430-bc072372-9db0-11e8-9cd3-caa809964ac1.png)

**Comment by Rayhane-mamah** [comment]
It will converge don't worry, I am no entirely sure why it became slower to align as I didn't pinpoint the exact time it started doing that but the current default parametes allow for best convergence so we're good. (Pre-trained LJspeech model coming in a couple of hours in case you need it)

**Comment by begeekmyfriend** [comment]
It still did not align at 17K steps, I will continue training. In the previous versions it comes to align in 10K steps . Really strange.

**Comment by Rayhane-mamah** [comment]
@begeekmyfriend yeah I am aware, in the model I will upload later, it aligned at somewhere around 32k steps. I am assuming it is due to the audio parameters (sample_rate, fft_size and trim parameters) I use currently. On the other side though, audio quality is good :)

**Comment by begeekmyfriend** [comment]
I do not think it is the audio parameters that make it slow to align for I have tried `fft_size = 2048` in the old version. The sample rate was 22050.
![step-5000-align](https://user-images.githubusercontent.com/6031938/43993009-9c3aafba-9db9-11e8-834b-6620a44b87c6.png)

**Comment by Rayhane-mamah** [comment]
I am not really sure then, I'll have to retrace back changes I made recently to pinpoint where this is happening. But like I said, compared to audio quality and prosodic performance, it's fine if the alignments are a bit late as long as they are learned.

**Comment by begeekmyfriend** [comment]
Bet on you~

**Comment by Rayhane-mamah** [comment]
Ah by the way I remembered, mels are [0, 4] it is most probably that! :) 
I forgot I made that change, it's definitely that!

**Comment by begeekmyfriend** [comment]
No, I also tried `symmetric_mels = True` it was the same result.

**Comment by Rayhane-mamah** [comment]
Hmmm nevermind then I'll look into it more when I get the time

**Comment by begeekmyfriend** [comment]
Why did you replace convolution with CBHG as post net in commit https://github.com/Rayhane-mamah/Tacotron-2/commit/87bedae047353550ff2610a430d16715cd187aa1? Did it improve the audio quality?

**Comment by Rayhane-mamah** [comment]
Yes, from what I have tried, CBHG performed best on post processing mels to linear. It performed better than encoder-like convolutions + RNN.

**Comment by begeekmyfriend** [comment]
It achieved alignment at 31K steps for my dataset, you are right, we should be patient with the training.
![step-31000-align](https://user-images.githubusercontent.com/6031938/44063400-e1688604-9f92-11e8-9cc0-a820f44ed193.png)

**Comment by begeekmyfriend** [comment]
@Rayhane-mamah You could shoot a paper on your discovery ;-)

**Comment by Rayhane-mamah** [comment]
You think? :) 
I prefer making a ground breaking paper or die trying haha.. No seriously, I currently am not in a favorable position for that, but sure, one day one might hope to contribute with something to the Deep Learning community :)
